js/demo-app/src/component.js

```diff
@@ -1,3 +1,6 @@
+import "@babel/polyfill";
+import "url-polyfill";
+import "nodelist-foreach-polyfill";
```

Comment:
url-polyfill was needed for both the demo and our lib while the foreach poly was just for the demo.  I think we should ultimately come up with a list of polyfills needed for the library and document them.

Perhaps, we should even offer a few different targets.  es6 modules, umd module, and umd/min modules with polyfills.  I have done this previously with rollup and it worked out pretty well.  Will just take adding a few different webpack module exports and some main entries in package.json.  But that's for another day.